docs/azure_jumpstart_arcbox/Full/_index.md

```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### Adventure Workload and Azure API Management
+
+This section guide you through deploy Adventure Work WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other ARC service. 
```

Comment:
Can you please check it is "Adventure Work" or "Adventure Works"? Also it is not ARC, it has to be Azure Arc. Standalone Arc is not recommended in our product documentation.